src/data/markdown/docs/03 cloud/01 Creating and running a test/04 Environment variables.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Environment variable'
+excerpt: 'How to use environment variable in the cloud'
+---
+
+Environment Variables are key-value pairs configured at the organization level and can be used to store information that you want to reference in your cloud test-scripts across all projects of this organization. All values are encrypted at rest and their plain value can only be visible to the owner and admins of the organization. All members of the organization can reference the environment variables in their test scripts. 
```

Comment:
I think we should elaborate on why environment variables are necessary for security reasons. Either here or in another section.



Describe how sensitive information can be accessible when transmitting the data or accessible from the k6 Cloud app.

https://docs.github.com/en/actions/security-guides/encrypted-secrets